### PR TITLE
[stable/home-assistant] OWNERS and convert configurator to sidecar

### PR DESCRIPTION
**Pull request by billimek**
**What this PR does / why we need it**:  

* Adds the OWNERS file to the chart
* Updates Home-Assistant image to 0.77.1
* Refactors the optional 'configurator' GUI to run as a sidecar instead of a completely independent deployment. The reason for the sidecar is that the configurator UI requires access to the home assistant configuration volume, and running it as an independent pod is an issue for situations where the underlying persistent storage is not shareable across pods (e.g. rook block storage)
* Switches to default ReadWriteOnce for PVC accessmode

### CODE REVIEW

File: stable/home-assistant/values.yaml [hunk 1]
```diff
@@ -50,7 +50,7 @@ persistence:
   ## If you want to reuse an existing claim, you can pass the name of the PVC using
   ## the existingClaim variable
   # existingClaim: your-claim
-  accessMode: ReadWriteMany
+  accessMode: ReadWriteOnce
```
Comment (unguiculus) [inline stable/home-assistant/values.yaml:53]: What would be the implications for multiple replicas? You'd probably need `ReadWriteMany` which is not supported by all environments anyways. How does the app behave with multiple replicas? Can it be run safely at all with a shared PV?

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/home-assistant/values.yaml (hunk 1 above)
Comment (billimek) [reply stable/home-assistant/values.yaml:53]: It really depends on how it is configured once deployed.  The default Home Assistant configuration is to leverage an on-disk sqllite database to persist state and other application level info.  With that, its probably best to stay at 1 replica.  The other config files are read once at startup time and wouldn't be impacted by multiple replicas.

It is, however, possible to configure Home Assistant to leverage an 'external database' like mariaDB to persist the state and historical information.  In this mode of operating it is possible to run with multiple replicas (I have tested this config myself).

The purpose of the change in this PR from ReadWriteMany to ReadWriteOnce is to support the change in how the optional sidecar Configurator accesses the configuration files.  With it being a sidecar, it is no longer necessary to run it as an external pod and there is no longer a requirement to have the PV access be ReadWriteMany.

**Review by unguiculus — APPROVED** [review]

**Pull request merged**